lib/active_model/serializer/attributes.rb

```diff
@@ -34,10 +34,11 @@ def inherited(base)
         #   class AdminAuthorSerializer < ActiveModel::Serializer
         #     attributes :id, :name, :recent_edits
         def attributes(*attrs)
+          options = attrs.pop if attrs.last.class == Hash
```

Comment:
I guess something like would be better:

```diff
+          options = attrs.last.class == Hash ? attrs.pop : {}
+          options = options.except(:key)
```

At least, we prevent ourselves from having unintended behavior.